autopush/websocket.py

```diff
@@ -163,42 +238,29 @@ def wrapper(result, *w_args, **w_kwargs):
     def base_tags(self):
         """Property that uses None if there's no tags due to a DataDog library
         bug"""
-        return self._base_tags if self._base_tags else None
+        return self.ps._base_tags if self.ps._base_tags else None
 
     def log_err(self, failure, **kwargs):
         """Log a twisted failure out through twisted's log.err"""
         log.err(failure, **kwargs)
 
-    def pauseProducing(self):
```

Comment:
Are `pauseProducing` and friends are only called by us, or can Twisted call them, too? (Say, if we're sending a batch of updates to a slow client).